app/src/main/res/layout/fragment_profile_list.xml

```diff
@@ -5,11 +5,28 @@
     android:layout_height="match_parent"
     android:orientation="vertical" >
 
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:fontFamily="@font/fredoka_bold"
+        android:padding="20dp"
+
+        android:text="@string/title_attendees_list"
+        android:textSize="24sp" />
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:fontFamily="@font/fredoka_normal"
+        android:text="@string/search_user_hint"
+        android:paddingHorizontal="20dp"
+        android:textSize="14sp" />
+
     <SearchView
         android:id="@+id/profile_list_search"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content" >
-
+        android:layout_height="wrap_content"
+        android:queryHint="Search for user">
```

Comment:
would it be possible to move this hint string to the strings so we can change the language ? That would be a wonderful add for foreign users, isn't it ?

Reply:
Sorry my bad, it was in strings I just didn't link it